app/src/main/java/edu/uc/jonesbr/myplantdiary/ui/main/MainFragment.kt

```diff
@@ -173,4 +168,8 @@ class MainFragment : Fragment() {
         }
     }
 
+    companion object {
+        fun newInstance() = MainFragment()
+    }
+
```

Comment:
Moved companion object to bottom, as specified here: https://kotlinlang.org/docs/reference/coding-conventions.html